releaf.gemspec

```diff
@@ -38,5 +38,4 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'spring-commands-rspec'
   s.add_development_dependency 'roo'
 
-  s.required_ruby_version = '>= 2.2.0'
```

Comment:
I disagree. releaf.gemspec is only for:
a) quickly adding all releaf-* gems
b) teting

And releaf gems depends on releaf-core gem, not releaf gem.
Rephrasing you: "mohter" of all releaf gems is **releaf-core**